xml/art_solution_automation.xml

```diff
@@ -166,6 +166,175 @@ https://confluence.suse.com/x/8IEnGw
+            </para>
+            <screen>&prompt.user;<command>wget</command> -O hana.sls \
+  https://github.com/SUSE/saphanabootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O netweaver.sls \
+  https://github.com/SUSE/sapnwbootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O drbd.sls \
+  https://github.com/SUSE/drbd-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O cluster.sls \
+  https://github.com/SUSE/habootstrap-formula/raw/master/pillar.example</screen>
+          </step>
+          <step>
+            <para>
+              Copy all pillar files to the right directory:
+            </para>
+            <screen>&prompt.sudo;<command>cp</command> -vi hana.sls cluster.sls /srv/pillar/hana
+&prompt.sudo;<command>cp</command> -vi netweaver.sls cluster.sls /srv/pillar/netweaver
+&prompt.sudo;<command>cp</command> -vi drbd.sls cluster.sls /srv/pillar/drbd</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Modify the content of the pillar files depending on your needed configuration
+          of each &salt; formula.
+        </para>
+        <para>
+          Notice, in this example, we have three different
+          <filename>cluster.sls</filename> files. Each might have different HA cluster
+          requirements.
+        </para>
+        <remark>toms 2020-09-23: Maybe we should link to the upstream Salt documentation here?
+        Or does the user need any other helpful tips, to watch out for errors etc.?
+        </remark>
+      </step>
+      <step>
+        <para>
+          Create the file <filename>/srv/salt/top.sls</filename> with the following content:
+        </para>
+        <screen>base:
+    'hana01,hana02':
+        - hana
+        - cluster
+
+    'netweaver01,netweaver02':
+        - netweaver
+        - cluster
+
+    'drbd01,drbd02':
+        - drbd
+        - cluster</screen>
+        <para>
+          In the above example, folder part is removed, as we are pointing to the formulas with
+          their name.
+          <remark>toms 2020-09-23: why? Could we make it consistent with /srv/pillar/top.sls?
+          </remark>
+        </para>
+      </step>
+      <step>
+        <para> Execute &salt; as shown in the following commands. Keep in mind,
+          &salt; supports more options, it is just an example: </para>
+        <stepalternatives>
+          <step>
+            <para>If you use a &saltmaster; with &salt; client, run:</para>
+            <screen>&prompt.root;<command>salt</command> '*' state.highstate</screen>
+          </step>
+          <step>
+            <para>If you use a &salt; client only, run:</para>
+            <screen>&prompt.root;<command>salt</command> --local state.highstate</screen>
+          </step>
+        </stepalternatives>
+        <remark>toms 2020-09-23: I miss some kind of "closing" step. How can I see/check
```

Comment:
The salt execution will report the status of the deployment. If everything is green (literally), everything went fine. If we have some state with error (in red), this means that something went wrong.

Reply:
Ahh great! :+1: I've replaced the remark with:

```
The command shows you the result (green for everything is fine, red for errors).
```